xt-prod-devel-rpi5-domd/recipes-bsp/xt-rpi-u-boot-scr/xt-rpi-u-boot-scr.bbappend

```diff
@@ -1,8 +1,11 @@
 require include/dt_config.inc
 
 XEN_OVERLAYS:append = "${@bb.utils.contains("MACHINE_FEATURES", "scmi", " ${XEN_DT_SCMI}.dtbo", "", d)}"
+XEN_OVERLAYS:append = "${@bb.utils.contains("MACHINE_FEATURES", "domd_wifi", " ${XEN_WIFI_PASSTHROUGH_DT_NAME}.dtbo", "", d)}"
 
 SCMI_PCIE1_DT_NAME_ADD = "${@bb.utils.contains("MACHINE_FEATURES", "domd_nvme", "${SCMI_DOMD_PCIE1_DT_NAME}.dtbo", "", d)}"
 
 DOMD_OVERLAYS:append = "${@bb.utils.contains("MACHINE_FEATURES", "scmi", " ${SCMI_DOMD_DT_NAME}.dtbo", "", d)}"
 DOMD_OVERLAYS:append = "${@bb.utils.contains("MACHINE_FEATURES", "scmi", " ${SCMI_PCIE1_DT_NAME_ADD}", "", d)}"
+
+DOMD_OVERLAYS:append = "${@bb.utils.contains("MACHINE_FEATURES", "domd_wifi", " ${DOMD_WIFI_DT_NAME}.dtbo", "", d)}"
```

Comment:
IMO, this approach for adding dtbo is getting out of hand. Lots of almost duplicated code....

Also, commit message clearly mentions `linux:` but you are making changes not only to linux.